frontend/components/docs/js/docs.service.ts

```diff
@@ -14,6 +15,8 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+import { selectIsMetadataVisible } from '../../../modules/viewer';
```

Comment:
@simplypixi @dleszcz any reason why we're not migrating this service?

Reply:
Nope - removed ;)